src/views/Dashboard.vue

```diff
@@ -182,29 +175,23 @@ export default {
       return (this.entityStatus === EntityStatus.FILED_INCORP_APP)
     },
 
-    /** Whether to block a new filing because another item has to be finished first.
-     * No changes are allowed if
-     * 1) this is a temporary reg number until it switches to a real business number
-     * 2) has a filing pending (is PAID) and waiting for completion
-     * 3) has a blocker task in the todo list (ie. draft, paid, error, correction )
+    /**
+     * Whether to block a new filing, ie:
+     * 1) there is a blocker task in the todo list or blocker filing in the filing history list
+     * 2) this is a temporary reg number until it switches to a real business number
      */
     disableChanges (): boolean {
-      return (this.hasBlockerTask || this.hasPendingFiling || !!this.tempRegNumber)
+      return (this.hasBlocker || !!this.tempRegNumber)
     },
 
     /** The Incorporation Application's Temporary Registration Number string. */
     tempRegNumber (): string {
       return sessionStorage.getItem('TEMP_REG_NUMBER')
-    },
-
-    /** True if StaffNotation menu should be rendered. */
-    visibleForStaff (): boolean {
-      return getFeatureFlag('staff-notation-enabled') && this.isRoleStaff
```

Comment:
This flag is now True in all environments, so I deleted it (from this project only -- I don't think it's used in Edit UI but that can be updated later).